.eslintrc.json

```diff
@@ -30,7 +30,6 @@
     "no-console": "warn",
     "no-plusplus": "error",
     "no-undef": "warn",
-    "object-curly-spacing": ["warn", "always"],
-    "sort-keys": ["warn", "asc", { "caseSensitive": true, "natural": true, "minKeys": 2 }]
```

Comment:
😿

Reply:
It doesnt have autofix, really annoying.. I've read some debate of removing it by devs also

Reply:
Yes, I will not object to removing this.